edx_sysadmin/views.py

```diff
@@ -117,8 +118,7 @@ class CoursesPanel(SysadminDashboardBaseView):
 
     def get_course_summaries(self):
         """Get an iterable list of course summaries."""
-
-        return modulestore().get_course_summaries()
+        return get_visible_courses()
```

Comment:
Now this will not produce summaries instead course overviews??
 https://github.com/eduNEXT/edx-platform/blob/master/lms/djangoapps/branding/__init__.py#L19

Reply:
yes, this will use course overviews since the application just uses course id and display name and both fields are in both models and are the same kind of field

Reply:
the RELEASE.rst file mentions that they are using course_summaries  for optimization and get_course_summaries() accepts the extra argument `org` however a site could have multiple orgs and a pass multiple orgs will call `get_course_summaries` multiple times and also I'm still not sure that `get_course_summaries` is better than `get_visible_courses`  since `get_visible_courses` makes a simple sql query and returns the courses in this case  but `get_course_summaries`  gets the courses from mongo and then it applies a specific logic for each course